src/server/enrollment-system.js

```diff
@@ -63,35 +63,97 @@ function veteranToPersonInfo(veteran) {
   };
 }
 
+function zeroPadNumber(number, padding) {
+  return (new Array(padding + 1).join('0') + number).slice(-padding);
+}
+
+function formDateToESDate(dateObject) {
+  if (dateObject.month >= 1 && dateObject.month <= 12 &&
+    dateObject.day >= 1 && dateObject.month <= 31 && // TODO: how robust does this need to be? Form validates as well.
```

Comment:
Re TODO: form validation is not trustable since anyone could conceivably post data to us.